forte/data/batchers.py

```diff
@@ -144,6 +145,132 @@ def default_configs(cls) -> Dict[str, Any]:
         raise NotImplementedError
 
 
+class FixedSizeDataPackBatcherWithExtractor(ProcessingBatcher):
+    def __init__(self, cross_pack: bool = True):
+        super().__init__(cross_pack=cross_pack)
+        self.instance_pool: List[Annotation] = []
+        self.feature_pool: List[Dict[str, Feature]] = []
+        self.pool_size = 0
+        self.batch_is_full = False
+
+    def initialize(self, config):
+        super().initialize(config)
+        self.scope = config["scope"]
```

Comment:
The default configs are `None` here, so it is good to check for not None

Reply:
Got it.